ui/src/app/job-list/table/table.component.ts

```diff
@@ -200,4 +200,26 @@ export class JobsTableComponent implements OnInit {
       this.jobs.forEach(j => this.selection.select(j));
     }
   }
+
+  updateCheckBoxSelection(clickedJob: QueryJobsResult, event: MouseEvent): void {
+    // if the user has shift-clicked on a job, find the last selected job and
+    // select all jobs between the two
+    if (event.shiftKey && this.selection.selected.length) {
+      const lastJobClicked = this.selection.selected[this.selection.selected.length - 1];
+      this.selection.select(clickedJob);
+      let indexes = [this.jobs.findIndex(j => j.id === lastJobClicked.id),
```

Comment:
very_optional: another (IMO) pretty cool way to write this, but perhaps more complicated:

```
const [from, to] = [
  this.jobs.findIndex(j => j.id === lastJobClicked.id),
  this.jobs.findIndex(j => j.id === clickedJob.id)
].sort();
```

Reply:
actually, I think that's clearer -- thanks!